lib/taskana-core/src/main/java/pro/taskana/impl/TaskServiceImpl.java

```diff
@@ -316,6 +316,16 @@ public TaskQuery createTaskQuery() {
         return (results == null) ? new ArrayList<>() : results;
     }
 
+    public boolean isThereATaskForThisClassificationKeyInTheseDomains(String classificationKey, String... domain) throws NotAuthorizedException {
```

Comment:
See other comments. I guess this need to be updated with naming and parameter checks.

Reply:
better don't even create a method in TaskService for deleting classification. The logic belongs to ClassificationService.